### PR TITLE
Added clarification

### PR DESCRIPTION
**Pull request by RichardLitt**
_first_ was ambiguous.

### CODE REVIEW

File: create-list.md [hunk 1]
```diff
@@ -4,6 +4,6 @@
 - Wait at least a couple of weeks after creating a list before submitting to this list to give it a chance to mature.
 - Search this list before making a new one, as yours may be a duplicate. If it is, try and contribute to the best one instead of making your own.
 - A link back to this list from yours, so users can discover more lists, would be appreciated. 
-- Make sure you read the [contribution guidelines](https://github.com/sindresorhus/awesome/blob/master/contributing.md) first.
+- Make sure you read the [contribution guidelines](https://github.com/sindresorhus/awesome/blob/master/contributing.md) before submitting a PR to this list to add your own list.
```
Comment (sindresorhus) [inline create-list.md]: > before submitting a PR to this list adding your own list.

Comment (RichardLitt) [inline create-list.md]: They're both grammatical?

Comment (aleksandar-todorovic) [inline create-list.md]: I suggest something like:

> ...before submitting a request for your list to be added here.

The whole `to this list to add your list` thingy sounds a bit confusing to me.

Comment (sindresorhus) [inline create-list.md]: Agreed.

Comment (RichardLitt) [inline create-list.md]: Noted and changed. Thanks for helping me out!

**Pull request merged**